frontend/src/app/modules/matches/matches-computations.js

```diff
@@ -17,9 +17,15 @@ export const generateMatchRatingChanges = (userId, userMatches) => userMatches.m
     }
 })
 
-export const generateRatingHistoryGraphForUser = (userMatches, userId, initialRating) => userMatches
-  .reverse()
-  .map(
+export const generateRatingHistoryGraphForUser = (userMatches, userId, initialRating) => {
+  const reversedUserMatches = userMatches.reverse()
+  const firstMatchDate = userMatches[0] ? userMatches[0].date : null
+  const graphStartDate = firstMatchDate ? new Date(firstMatchDate) : new Date()
+  if (firstMatchDate) {
+    graphStartDate.setDate(firstMatchDate.getDate() - 1)
+  }
+  return [{ x: graphStartDate, y: initialRating }]
```

Comment:
cannot we do it without concat? 

return [{ x: graphStartDate, y: initialRating }, ...evaluateMatchesInTime(reversedUserMatches)]

Reply:
I'd prefer not to. I believe the method with `concat` is faster. I don't like overusing spreading. However, if you put your foot down and say "I AM THE CODE OWNER, OBEY ME", I'll change it.

Reply:
leaving my foot up for on this one:)